src/lib/field.test.ts

```diff
@@ -4,7 +4,7 @@ describe('Field constructor', () => {
   beforeAll(() => isReady);
```

Comment:
We have a `field.test.ts` and a `field.unit-test.ts`, although one uses Jest, I think we should consider combining both into `field.unit-test.ts`

Reply:
yeah you're right, the old test does almost nothing